XMonad/Actions/CopyWindow.hs

```diff
@@ -76,18 +76,20 @@ import qualified XMonad.StackSet as W
 -- "XMonad.Doc.Extending#Editing_key_bindings".
 
 -- $logHook
--- To distinguish workspaces containing copies of the focused window use
--- something like:
---
--- > sampleLogHook h = do
--- >    copies <- wsContainingCopies
--- >    let check ws | ws `elem` copies = pad . xmobarColor "red" "black" $ ws
--- >                 | otherwise = pad ws
--- >    dynamicLogWithPP myPP {ppHidden = check, ppOutput = hPutStrLn h}
--- >
--- > main = do
--- >    h <- spawnPipe "xmobar"
--- >    xmonad def { logHook = sampleLogHook h }
+-- To distinguish workspaces containing copies of the focused window, use the functions below.
+-- First use a check copy function that takes a pretty printer and makes it aware of copies.
+--
+-- > copiesPP :: PP -> X PP
+-- > copiesPP pp = do
+-- >     copies <- wsContainingCopies
+-- >     let check ws | ws `elem` copies = xmobarColor orange "" ws 
+-- >                  | otherwise = xmobarColor yellow "" ws
+-- >     return pp { ppHidden = check }
+-- 
+-- Then use @copiesPP@ with 'statusBarProp':
```

Comment:
You need to do 'XMonad.Hooks.StatusBar.statusBarProp' here, otherwise I don't think Haddock will link that (since the module itself does not import it). 

Might also be worth to explicitly import it, not sure

Reply:
Thanks!!! I just ignore it.